### PR TITLE
Support Markdown rendering for vulnerability details.

### PR DESCRIPTION
**Pull request by oliverchang**
image

### CODE REVIEW

File: gcp/appengine/frontend3/src/templates/vulnerability.html [hunk 1]
```diff
@@ -27,7 +27,10 @@ <h1 class="title">{{ vulnerability.id }}</h1>
         <dt>Modified</dt>
         <dd>{{ vulnerability.modified }}</dd>
         <dt>Details</dt>
-        <dd><pre class="details">{{ vulnerability.details }}</pre></dd>
+        <!-- <dd><pre class="details">{{ vulnerability.details }}</pre></dd> -->
```
Comment (rzhw) [inline gcp/appengine/frontend3/src/templates/vulnerability.html:30]: Probably can remove this?

File: gcp/appengine/frontend3/src/styles.scss [hunk 2]
```diff
@@ -422,6 +422,53 @@ dl.vulnerability-details,
     grid-column: 2;
   }
 
+
+  dd.details {
+    overflow-wrap: break-word;
+    li {
```
Comment (rzhw) [inline gcp/appengine/frontend3/src/styles.scss:428]: I wonder if the styles here and below we should be pulling out into a more top-level position.

**Review by rzhw — CHANGES_REQUESTED** [review]

File: gcp/appengine/frontend3/src/templates/vulnerability.html (hunk 1 above)
Comment (oliverchang) [reply gcp/appengine/frontend3/src/templates/vulnerability.html:30]: Good catch, removed!

File: gcp/appengine/frontend3/src/styles.scss (hunk 2 above)
Comment (oliverchang) [reply gcp/appengine/frontend3/src/styles.scss:428]: Hmm, this is to set specific styles for the user provided markdown, which probably don't make sense to be more top level? 

Do you have a suggestion for how to make these all cleaner?

Comment (rzhw) [reply gcp/appengine/frontend3/src/styles.scss:428]: Good point - I felt like I saw duplication but when I took a closer look it's just this bit that I'm concerned with:

```css

    li {
      $active-spacing: 6px;
      margin: -$active-spacing 0;
      padding: $active-spacing 0;
    }
```

It's duplicated from `.top-bar` - this makes sense in a horizontal list of links context but doesn't seem to be styling that makes sense for typical paragraph content?

**Review by rzhw — APPROVED** [review]

File: gcp/appengine/frontend3/src/styles.scss (hunk 2 above)
Comment (oliverchang) [reply gcp/appengine/frontend3/src/styles.scss]: removed as discussed!

**Pull request merged**